docs/install.rst

```diff
@@ -1,13 +1,53 @@
 =====================
-Quick Installation
+Get Certbot
 =====================
 
```

Comment:
Now that this file is a lot longer, perhaps we should add a table of contents to the top? You can do this by adding:
```
.. contents:: Table of Contents
   :local:
```
underneath the title as seen in some of the other `.rst` files.

If you make this change, I think we should also give a title to the first section starting with "Certbot is packaged for...". Without it, the docs look like this:
![screenshot from 2016-10-25 15-32-51](https://cloud.githubusercontent.com/assets/6504915/19706724/54f596ce-9ac8-11e6-8ee1-cb1e9b6e1c4e.png)

To me, this looks a little weird or at the very least not consistent with our other files. I don't have a strong opinion here though, so feel free to do what you think works best.